### PR TITLE
Adding nanoFramework.Runtime.Native to NuSpec

### PR DESCRIPTION
**Pull request by CoryCharlton**
## Description
- Adding nanoFramework.Runtime.Native to NuSpec

## Motivation and Context
I was having issues running benchmarks with the latest version. This doesn't resolve them but it still seems like something that should be fixed.

## How Has This Been Tested?
- N/A

## Types of changes

- [ ] Improvement (non-breaking change that improves a feature, code or algorithm)
- [ ] Bug fix (non-breaking change which fixes an issue with code or algorithm)
- [ ] New feature (non-breaking change which adds functionality to code)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Config and build (change in the configuration and build system, has no impact on code or features)
- [X] Dependencies (update dependencies and changes associated, has no impact on code or features)
- [ ] Unit Tests (add new Unit Test(s) or improved existing one(s), has no impact on code or features)
- [ ] Documentation (changes or updates in the documentation, has no impact on code or features)

## Checklist:

- [ ] My code follows the code style of this project (only if there are changes in source code).
- [ ] My changes require an update to the documentation (there are changes that require the docs website to be updated).
- [ ] I have updated the documentation accordingly (the changes require an update on the docs in this repo).
- [X] I have read the [CONTRIBUTING](https://github.com/nanoframework/.github/blob/main/CONTRIBUTING.md) document.
- [ ] I have tested everything locally and all new and existing tests passed (only if there are changes in source code).
- [ ] I have added new tests to cover my changes.

### CODE REVIEW

**Review by torbacz — APPROVED** [review]

File: nanoFramework.Benchmark.nuspec [hunk 1]
```diff
@@ -20,6 +20,7 @@
     <dependencies>
       <dependency id="nanoFramework.CoreLibrary" version="1.15.5" />
       <dependency id="nanoFramework.Logging" version="1.1.74" />
+	  <dependency id="nanoFramework.Runtime.Native" version="1.6.12" />
```
Comment (Ellerbach) [inline nanoFramework.Benchmark.nuspec:23]:
```suggestion
      <dependency id="nanoFramework.Runtime.Native" version="1.6.12" />
```

**Review by Ellerbach — COMMENTED** [review]
Is it really necessary to add?

File: nanoFramework.Benchmark.nuspec (hunk 1 above)
Comment (torbacz) [reply nanoFramework.Benchmark.nuspec:23]: It's in csproj, so It also should be in nuspec

**Comment by josesimoes** [comment]
This is not required at all.
To clear any doubts: these checks were added to our Versioncop tool 😉 
https://dev.azure.com/nanoframework/nanoFramework.Benchmark/_build/results?buildId=45036&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=1a5cc010-7735-550a-9d76-c0b745122dab&l=41

**Pull request closed without merging**